pyproject.toml

```diff
@@ -6,7 +6,10 @@ build-backend = "hatchling.build"
 name = "jupyter_ai_monorepo"
 dynamic = ["version", "description", "authors", "urls", "keywords"]
 requires-python = ">=3.8"
-dependencies = []
+dependencies = [
+    "jupyter-ai-magics @ {root:uri}/packages/jupyter-ai-magics",
+    "jupyter-ai @ {root:uri}/packages/jupyter-ai"
+]
```

Comment:
Generally this is required for `pip install .` to work from top-level directory. Otherwise the `pyproject.toml` does not really constitute a monorepo project, but just some set of settings for it.

Specifically, this is also required for the `minimum` `dependency-type`  to work (once https://github.com/jupyterlab/maintainer-tools/pull/250 is in) but other actions which support monorepo structure also assume that this exists.

Reply:
Ah, I see. Yes, running `pip install .` would definitely make more sense than running `jlpm dev-install` in CI.